installing/installing_restricted_networks/installing-restricted-networks-preparations.adoc

```diff
@@ -5,38 +5,69 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-Before you install a cluster on infrastructure that you provision in a restricted network, you must create a mirror registry. Installations on a restricted network are supported on only infrastructure that you provision, not infrastructure that the installer provisions.
+Before you install a cluster on infrastructure that you provision in a restricted network, you must mirror the required container images into that environment. Installations on a restricted network are supported on only infrastructure that you provision, not infrastructure that the installer provisions. You may also use this procedure in unrestricted networks to ensure your clusters only use container images that have satisfied your organizational controls on external content.
 
 [IMPORTANT]
 ====
-You must have access to the internet to obtain the data that populates the mirror
-repository. In this procedure, you place the mirror registry on a bastion host
+You must have access to the internet to obtain the necessary container images.
+In this procedure, you place the mirror registry on a mirror host
 that has access to both your network and the internet. If you do not have access
-to a bastion host, use the method that best fits your restrictions to bring the
-contents of the mirror registry into your restricted network.
+to a mirror host, use the disconnected procedure to copy images to a device you
+can move across network boundaries with.
 ====
 
 include::modules/installation-about-mirror-registry.adoc[leveloffset=+1]
 
 [id="installing-preparing-bastion"]
-== Preparing the bastion host
+== Preparing your mirror host
```

Comment:
I'd like to not use bastion because I think with disconnected there is no bastion, so terminology is too vague.  I tried a couple, 'mirror host' sounds reasonable.

Reply:
Can we update the earlier `[id="installing-preparing-bastion"]` -> `[id="installing-preparing-mirror"]`?  Or maybe inclide `[id="..."]` for both forms for backwards-compat?